src/Sylius/Behat/MultiContainerExtension/ContextRegistry.php

```diff
@@ -40,7 +40,11 @@ public function add($serviceId, $serviceClass)
     public function getClass($serviceId)
     {
         if (!isset($this->registry[$serviceId])) {
-            throw new \InvalidArgumentException();
+            throw new \InvalidArgumentException(sprintf(
+                'Could not find class for service with id "%s". Make sure you tagged the context with "%s".',
+                $serviceId,
+                'sylus.behat.context'
```

Comment:
Well, not a big deal for me, but still I think it is pointless. Not really more readable. I always assume that sprintf arguments are something dynamic.